language-support/ts/codegen/tests/ts/generated/src/__tests__/test.ts

```diff
@@ -65,38 +66,49 @@ afterAll(() => {
   console.log('Killed JSON API');
 });
 
+// TODO(MH, #3518): `daml2ts` should generate this type as `Main.Person.Key`.
+type PersonKey = {_1: daml.Party; _2: daml.Int}
+
 test('create + fetch & exercise', async () => {
   const ledger = new Ledger(ALICE_TOKEN, `http://localhost:${JSON_API_PORT}/`);
-  const alice: Main.Person = {
+  const alice5: Main.Person = {
     name: 'Alice from Wonderland',
     party: ALICE_PARTY,
     age: '5',
   };
-  const aliceKey = {_1: alice.party, _2: alice.age};
-  const aliceContract = await ledger.create(Main.Person, alice);
-  expect(aliceContract.payload).toEqual(alice);
-  expect(aliceContract.key).toEqual(aliceKey);
+  const alice5Key = {_1: alice5.party, _2: alice5.age};
```

Comment:
I'm planning to add another birthday via `exerciseByKey`. That's why I added the 5 and 6 everywhere.